docs/presets.rst

```diff
@@ -153,6 +153,15 @@ it into an MP4 container:
 
 * ``best_video_quality``
 
+
+Max 1080p
+^^^^^^^^^^^^^^^^^^
+
+Add the following preset to download the best available (but not greater then 1080P video) and audio quality
```

Comment:
Last nit: "(but not greater than) 1080p video and audio quality"

Reply:
If I want to nit, audio quality can't be 1080p. so having the 1080p limit in parentheses I believe would be the most accurate.
It is still the best audio quality. Only the video quality is limited.

Reply:
True, how about 'best available audio and video quality, with the video not greater than 1080p'